sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/SecHubRemoteCredentialContainer.java

```diff
@@ -0,0 +1,36 @@
+package com.mercedesbenz.sechub.commons.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SecHubRemoteCredentialContainer {
```

Comment:
Hmm... "container" means normally something "big" which hold child data etc. and gives the possibility for something like "getChildren()" or any additional methods. Oft containers acts like a list.

Here we have only data for credential situation which itself is always given back inside a list by the support.

I would suggest here a name change:

```suggestion
public class SecHubRemoteCredentialData {
```
or

```suggestion
public class SecHubRemoteCredentialEntry {
```

Means: The SecHubRemoteCredentialConfiguration has "credentials" field which contains data/entry elements.